File: python/e10.branch.py
#!/usr/bin/python
# get input from user
response = raw_input("Would you like a toast? [Yes/No]: ")
# set response string using one line
#   Test response to a string
response_string = "That's great" if response == "Yes" else "How about a muffin?"
# output the response string
print response_string

File: lessons/gen_scripts/ruby/scripts/e10.branch.rb
#!/usr/bin/env ruby
print "Would you like a toast? [Yes/No]: " # print prompt
$stdout.flush                              # flush unbuffered stream
response = gets.chomp                      # acquire string input (remove newline)

# set response string using ternary (single-line) conditional
response_str = response == "Yes" ? "That's great!" : "How about a muffin?"

puts response_str
